docs/android/gradle.md

```diff
@@ -78,39 +93,40 @@ If the project is configured to prefer settings repositories maven source should
         }
     }
     ```
+Note: If you use `RepositoriesMode.FAIL_ON_PROJECT_REPOS` mode you may experience a failure when one or more of the project's modules define their own repositories in their `build.gradle` files. Switching the mode to `RepositoriesMode.PREFER_SETTINGS` may solve this problem. Alternatively, you may add the productscience maven url to these module `build.gradle` files. You can also use the method defined in Case 2 below instead of defining them in `settings.gradle`.
```

Comment:
```suggestion
> **Note**: If you use `RepositoriesMode.FAIL_ON_PROJECT_REPOS` mode you may experience a failure when one or more of the project's modules define their own repositories in their `build.gradle` files.
> Switching the mode to `RepositoriesMode.PREFER_SETTINGS` may solve this problem.
> Alternatively, you may add the productscience maven url to these module `build.gradle` files.
> You can also use the method defined in Case 2 below instead of defining them in `settings.gradle`.
```